extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongoClientConfig.java

```diff
@@ -68,6 +68,12 @@
     @ConfigItem
     public List<String> hosts;
 
+    /**
+     * Configure the database name.
+     */
+    @ConfigItem
+    public Optional<String> database;
```

Comment:
Going this way, do we want to produce the MonoData and reactive equivalent directly?

Reply:
You propose to produce a MongoDatabase and equivalent reactive objects inside the bean container if the property is set in order the user to directly use it ?

Yes, this can be convenient, as it will always be the first thing a user will do : get the database, so we can do it for him.